src/Formatter/Http/SimpleJson.php

```diff
@@ -16,13 +16,15 @@
 final class SimpleJson implements Http
 {
     private Encoder $encoder;
+    private int $errorStatus;
 
     /**
      * @param Encoder $encoder
      */
-    public function __construct(Encoder $encoder)
+    public function __construct(Encoder $encoder, int $errorStatus = 500)
```

Comment:
Same here - would be better to use the Symfony constant if it's available in the bundle